src/dashboard/src/components/advanced_search.py

```diff
@@ -59,7 +71,9 @@ def search_parameter_prep(request):
             except:
                 types.insert(index, '')
 
-            index = index + 1
+        for index, field in enumerate(fields):
```

Comment:
Is this supposed to be inserting into ```fields``` after the above for loop?  Are we overwriting entries in ```fields```?   ```enumerate``` takes a second param of where to start indexing from.